packages/vertica-nodejs/lib/connection.js

```diff
@@ -146,6 +154,24 @@ class Connection extends EventEmitter {
             return self.emit('error', err)
           }
         }
+        // else if (self.tls_mode === 'prefer') { // basic TLS connection, does not verify CA certificate
```

Comment:
Looks like some cleanup can be done here

Reply:
Removed the code block in the latest commit.